src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -127,7 +127,7 @@ public void ValidPathWithTrailingSlash()
         }
 
         [ConditionalFact(nameof(UsingNewNormalization))]
-        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot)]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)]
```

Comment:
Are all of these also skipped because of AppContainer? If so, then can we add a message?